xtext-website/_posts/releasenotes/2019-06-04-version-2-18-0.md

```diff
@@ -17,26 +17,29 @@ Xtend has learned try-with-resources and the ternary operator. See the [Xtend Re
 
 ## Credits
 
-TODO update the list
-
 The Xtext project is thankful for the dedication of each committer and contributor. This release has been made possible by the following persons (in order of the number of contributed commits to this release to [all repositories](https://github.com/eclipse/xtext#repositories) except [xtext-xtend](https://github.com/eclipse/xtext-xtend)):
 
 - Christian Dietrich (itemis)
 - Karsten Thoms (itemis)
 - Sebastian Zarnekow (itemis)
-- Tamas Miklossy (itemis)
+- Jan Koehnlein (TypeFox)
 - Holger Schill (itemis)
 - Arne Deutsch (itemis)
-- Prajwal Gowda (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Anders Dahlberg (Ericsson) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Eva Poell (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Lorenzo Addazi ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Jan Rosczak ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Jelle Schuhmacher (Altran) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
-- Titouan Vervack (Sigasi) 
-- Jonathan Menzies ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
 - Christian Schneider (TypeFox)
-
+- Titouan Vervack (Sigasi)
+- Max Leuthäuser (itemis) ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- Akos Kitta (TypeFox)
+- Lorenzo Addazi
+- Lorenzo Bettini (Università degli Studi di Firenze)
+- Tamas Miklossy (itemis)
+- Dennis Huebner (TypeFox)
+- RhiobeT ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- Sergio Otero Lopez ![](https://img.shields.io/badge/-first%20time%20contributor-green.svg)
+- Eva Poell (itemis)
```

Comment:
Eva is no longer at itemis.

Reply:
At the time she was working on the feature she was working for itemis. But we may remove that here to reflect the current state. I'll do this with a separate change.